hack/set_containerImage.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash -e
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# Set the annotation `containerImage` in
+# bundle/manifests/vault-secrets-operator.clusterserviceversion.yaml to the
+# operator image in the deployment
+
+HACK_DIR=$(dirname "$0")
```

Comment:
nit: we could replace the call to dirname with bash's string manipulation: `HACKDIR="${0%/*}"`